geode-core/src/main/java/org/apache/geode/distributed/internal/tcpserver/TcpServer.java

```diff
@@ -381,6 +382,13 @@ private void processRequest(final Socket socket) {
         // client went away - ignore
       } catch (CancelException ignore) {
         // ignore
+      } catch (SocketTimeoutException ex) {
+        String sender = null;
+        if (socket != null) {
+          sender = socket.getInetAddress().getHostAddress();
+        }
+        // Do not want the full stack trace to fill up the logs
+        log.info("Exception in processing request from " + sender + ": " + ex.getMessage());
```

Comment:
Only feedback is that if the socket was null, then we will log: "Exception in processing request from null: <message>".  Might be better to log: "Exception in processing request from unknown sender: <message>".  Then again, I'm not sure under what conditions the socket would be null since we are catching a SocketTimeoutException here which implies a non-null socket.